v1.1/admin_ui_jobs_page.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Jobs Page
+toc: false
+feedback: false
+---
+
+The Jobs page of the Admin UI provides details of the backup and restore jobs, as well as schema changes performed across all nodes in the cluster.
+
+<div id="toc"></div>
+
+## Access the Jobs Page
+1. [Access the Admin UI](explore-the-admin-ui.html#access-the-admin-ui)
+2. In the left-hand navigation bar, click **Jobs**.
+
+The Jobs page appears.
+<img src="{{ 'images/admin_ui_jobs_page.png' | relative_url }}" alt="CockroachDB Admin UI Jobs Page" style="border:1px solid #eee;max-width:100%" />
+
+The Jobs table displays the user, description, creation time, and status of each backup and restore job, as well as schema changes performed across all nodes in the cluster.
```

Comment:
maybe move the columns to a second sentence?  It seems like the info to lead with is what jobs are shown, and then after that the details we display for each job.